src/it/java/org/checkstyle/suppressionxpathfilter/XpathRegressionUpperEllTest.java

```diff
@@ -64,4 +64,29 @@ public void testUpperEll() throws Exception {
                 expectedXpathQueries);
     }
 
+    @Test
+    public void testUpperEllDefault() throws Exception {
+        final File fileToProcess =
+            new File(getPath("SuppressionXpathRegressionUpperEllDefault.java"));
```

Comment:
`Default` - the first test also uses default config, so it's a little confusing here. Please rename input files and tests.
There are no properties in this Check, so let's just use numbers at the end:
```
SuppressionXpathRegressionUpperEll.java --> SuppressionXpathRegressionUpperEllFirst.java
SuppressionXpathRegressionUpperEllDefault.java -->  SuppressionXpathRegressionUpperEllSecond.java
```

Reply:
Changes Done